tests/input_set.py

```diff
@@ -135,6 +135,116 @@ def create_geometry(self):
         super(PinCellInput, self).create_geometry()
 
 
+class GridInput(InputSet):
+    """Sets up a simple grid problem using explicit plane"""
+
+    def create_materials(self):
+        """Instantiate C5G7 Materials."""
+        self.materials = \
+            openmoc.materialize.load_from_hdf5(filename='c5g7-mgxs.h5',
+                                               directory='../../sample-input/')
+
+    def create_geometry(self):
+        """Instantiate a 3x3 grid Geometry."""
+
+        # Create the planes that bound the cell and geometry
+        xplanes = [None] * 4
```

Comment:
If you like the pre-allocation then keep it. Alternatively, consider something more functional:

```python
xplanes = map(lambda x: openmoc.XPlane(x=x), [-3, -1, +1, +3])
yplanes = map(lambda x: openmoc.YPlane(y=y), [-3, -1, +1, +3])
```